task/01-strings-tasks.js

```diff
@@ -238,7 +252,7 @@ function encodeToRot13(str) {
  *   isString(new String('test')) => true
  */
 function isString(value) {
-    throw new Error('Not implemented');
+    return (typeof value === 'string' || value instanceof String) ? true : false;
```

Comment:
There are is no need need to use ternary operator here, in this case [logical operator OR](https://javascript.info/logical-operators) will return boolean value:)